### DISCUSSION TITLE
What is the preset for alphatab metronome sound inside the base soundfont?

### DISCUSSION THREAD

**Discussion by MDoder97** [CoderLine/alphaTab#553] [Q&A] [answered]
I'm using multiple soundfonts, and when I don't load the default soundfont (on only nylon guitar songs etc.) metronome doesn't work. I tried to find it but there doesn't seem to be a metronome preset in polyphone.

As a side question, how would I add a new metronome sound from a actual sound file, is this possible? like wav file etc.

Thanks.

**Comment by Danielku15** [comment, marked as answer]
alphaTab is using the [percussion channel](https://github.com/CoderLine/alphaTab/blob/5bc383743151996f77e6cdbab827601c0d04105f/src/synth/synthesis/TinySoundFont.ts#L698) with [note key 33](https://github.com/CoderLine/alphaTab/blob/5bc383743151996f77e6cdbab827601c0d04105f/src/synth/synthesis/TinySoundFont.ts#L119). 

Adding metronome via a sound file is a not supported feature as of today. You could try to use the feature for [handling midi events](https://alphatab.net/docs/guides/handling-midi-events) to build your own metronome if you solve the challenge of handling the audio latency.
